operations/agent-static-mixin/dashboards.libsonnet

```diff
@@ -286,6 +286,7 @@ local template = grafana.template;
             value: '$__all',
           },
           includeAll=true,
+          allValues='.*',
```

Comment:
I know I suggested you `.*` but `.+` may actually be slightly more accurate. That's also what `jsonnet-libs` does by default.

Reply:
Yeah, let's change it. I had both `addMultiTemplate` and `addMultiTemplateWithAll` helpers with a different default, changing it now.